src/Frontend.elm

```diff
@@ -800,7 +800,7 @@ codeOfConductContent =
 # Code of Conduct
 
 Elm Camp welcomes people with a wide range of backgrounds, experiences and knowledge. We can learn a lot from each other. It's important for us to make sure the environment where these discussions happen is inclusive and supportive. Everyone should feel comfortable to participate! The following guidelines are meant to codify these intentions.
-
+<br/>
```

Comment:
I think you might need to put 2 `<br />` in a row to get the extra gap you are after.

Maybe we could instead find the place where styles are being applied to the elements and display them in more consistent format. I had a quick look and I _think_ the bigger gaps are being generated because it's the bottom of a list. But this might prove tricky.